src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -276,6 +275,29 @@ export default class BrowserCell extends Component {
       });
     }
 
+    const validScripts = (this.props.scripts || []).filter(script => script.classes?.includes(this.props.className));
+    if (validScripts.length) {
+      onEditSelectedRow && contextMenuOptions.push({
+        text: 'Scripts',
+        items: validScripts.map(script => {
+          return {
+            text: script.title,
+            callback: async () => {
+              try {
+                const object = Parse.Object.extend(this.props.className).createWithoutData(this.props.objectId);
+                await Parse.Cloud.run(script.cloudCodeFunction, {object: object.toPointer()}, {useMasterKey: true});
+                this.props.showNote(`${script.title} ran with object ${object.id}}`);
```

Comment:
```suggestion
                const response = await Parse.Cloud.run(script.cloudCodeFunction, {object: object.toPointer()}, {useMasterKey: true});
                this.props.showNote(response || `${script.title} ran with object ${object.id}}`);
```

Would that work, so it outputs the response if there is one? I assume the response is `undefined` if the Cloud Code function just returns?

Reply:
Nice!